hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/hedera/embedded/AbstractEmbeddedHedera.java

```diff
@@ -192,8 +196,16 @@ public void start() {
                 .addressBook(toPbjAddressBook(addressBook))
                 .build());
         ((CommittableWritableStates) writableStates).commit();
+
+        final var writableRosterStates = state.getWritableStates(RosterService.NAME);
```

Comment:
⚠️ If this is necessary, there may be a bug somewhere---if the roster lifecycle isn't enabled, there should not be any code that depends on `RosterService` having adopted an active roster.

(But note [this PR](https://github.com/hashgraph/hedera-services/pull/17070) enables `rosterLifecycle=true` for all PR checks, so this _certainly_ isn't necessary given that work.)

Reply:
Removed this diff and now only `DabEnabledUpgradeTest` is failing but I don't think it is related to this one.